web_timeline/static/src/js/timeline_canvas.js

```diff
@@ -0,0 +1,128 @@
+/* Copyright 2018 Onestein
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define('web_timeline.TimelineCanvas', function (require) {
+    "use strict";
+    var Widget = require('web.Widget');
```

Comment:
Odoo 13.0 allows ES6, check where can use it.
```suggestion
    const Widget = require('web.Widget');
```